src/dynamics/solver/joint_constraint/prismatic_velocity_constraint.rs

```diff
@@ -179,35 +210,48 @@ impl PrismaticVelocityConstraint {
         /*
          * Setup limit constraint.
          */
-        let mut limits_forcedirs = None;
+        let limits_forcedir2 = axis2.into_inner(); // hopefully axis1 is colinear with axis2
```

Comment:
The old code used two different axis here, which means the applied impulses were not opposite and equal, thus violating conservation of momentum. When the constraint is closed to satisfied `axis1≈axis2`, and the choice here is arbitrary. If the angle constraint is very violated, then there is no single vector that makes sense here, but one could consider taking the mean of axis1 and axis2 here, just so as not to favor one constraint over the other. For now I think this is an improvement over what already existed, since at least we conserve momentum.